pkg/vsphere/rbac/rbac.go

```diff
@@ -128,13 +125,11 @@ func (am *AuthzManager) PrincipalBelongsToGroup(ctx context.Context, group strin
 
 	components := strings.Split(am.Principal, "@")
 	var domain string
-	name := components[0]
-	if len(components) < 2 {
-		domain = ""
-	} else {
+	if len(components) >= 2 {
 		domain = components[1]
 	}
 
+	name := components[0]
```

Comment:
nit: I think I'd find this clearer if left between the `components :=` line and the `var domain string` line. (Then the flow is: split, use the first element, use the second element if one exists.)

Reply:
That makes sense. I'll change it back.